services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/assets/[asset]/index.js

```diff
@@ -278,14 +278,16 @@ const Asset = ({ libraryData, params }) => {
                       View library assets
                       <ArrowRight size={16} />
                     </Button>{' '}
```

Comment:
Should this space still be there if the button next to it is no longer present? I didn't check though so maybe display-wise it doesn't make a difference?

Reply:
Doesn't appear to make a difference.

Reply:
👍